Tests/MapLibreSwiftUITests/Models/MapCamera/CameraStateTests.swift

```diff
@@ -1,34 +1,33 @@
 import CoreLocation
+import SnapshotTesting
 import XCTest
 @testable import MapLibreSwiftUI
 
 final class CameraStateTests: XCTestCase {
+    let coordinate = CLLocationCoordinate2D(latitude: 12.3, longitude: 23.4)
+
     func testCenterCameraState() {
-        let expectedCoordinate = CLLocationCoordinate2D(latitude: 12.3, longitude: 23.4)
-        let state: CameraState = .centered(onCoordinate: expectedCoordinate)
-        XCTAssertEqual(state, .centered(onCoordinate: CLLocationCoordinate2D(latitude: 12.3, longitude: 23.4)))
-        XCTAssertEqual(
-            String(describing: state),
-            "CameraState.centered(onCoordinate: CLLocationCoordinate2D(latitude: 12.3, longitude: 23.4)"
-        )
+        let state: CameraState = .centered(onCoordinate: coordinate)
+        XCTAssertEqual(state, .centered(onCoordinate: coordinate))
```

Comment:
Note that I switched all these unit tests to snapshot tests; will keep better separation between code and test fixtures, and makes it easier to add new cases. All we're doing is ensuring we catch changes anyways.